src/packages/core/modal/token/change-password-modal.token.ts

```diff
@@ -1,9 +1,19 @@
 import { UmbModalToken } from '@umbraco-cms/backoffice/modal';
 
 export interface UmbChangePasswordModalData {
-	requireOldPassword: boolean;
+	userId: string;
+	requireOldPassword?: boolean;
 }
 
-export const UMB_CHANGE_PASSWORD_MODAL = new UmbModalToken<UmbChangePasswordModalData>('Umb.Modal.ChangePassword', {
-	type: 'dialog',
-});
+export interface UmbChangePasswordModalValue {
+	newPassword: string;
+	confirmPassword: string;
+	oldPassword?: string;
```

Comment:
Could this just be one password?  on open its the current, on submit it's the new.
Then its the dialogs responsibility to handle the `confirmPassword`? So that's just UI of the modal, that the outsides does not know about?

Reply:
Ah, yes... We will have to send the old password back too.  It is needed when we change the password for the current user.